internal/host/transition_test.go

```diff
@@ -4029,8 +4053,18 @@ var _ = Describe("Refresh Host", func() {
 					expectedSeverity = models.EventSeverityWarning
 				}
 				if !t.errorExpected && srcState != t.dstState {
-					mockEvents.EXPECT().AddEvent(gomock.Any(), *host.ClusterID, &hostId, expectedSeverity,
-						gomock.Any(), gomock.Any())
+					// Note: without this condition there is no meaning to lines 3881-3884
```

Comment:
line numbers might be changed in the future (as code evolves), pls rephrase the comment.

Reply:
solved